app/serializers/export/comment_serializer.rb

```diff
@@ -0,0 +1,8 @@
+module Export
+  class CommentSerializer < ActiveModel::Serializer
+    attributes :text,
+               :author_id,
```

Comment:
Heh no wait, this is of course the users own comments, so no need to have a handle. BUT we need to say what post it relates to. How about building an url to the post?